packages/cli/src/commands/inspect/index.ts

```diff
@@ -192,6 +192,23 @@ async function printDetails({
   print(chalk.bold('  General\n\n'));
   print(`    ${chalk.cyan('id')}\t\t${id}\n`);
   print(`    ${chalk.cyan('name')}\t${name}\n`);
+  // @ts-ignore - customEnvironment needs to be defined on Deployment
+  const customEnvironmentName = deployment.customEnvironment?.name;
+  const target = customEnvironmentName ?? deployment.target;
+  if (target) {
+    print(`    ${chalk.cyan('target')}\t`);
+    // TODO: once custom environments is shipped for all users,
+    // make all deployments link to the environment settings page
+    print(
+      // @ts-ignore - customEnvironment needs to be defined on Deployment
+      customEnvironmentName && deployment.team?.slug
+        ? `${link(
```

Comment:
I think the consensus we had regarding links is that if the terminal does not support them, don't render the link. That is to say, we should add a fallback (2nd param of the `link()` function) which returns only the name of the target.